include/SFML/Window/Mouse.hpp

```diff
@@ -53,8 +53,8 @@ class SFML_WINDOW_API Mouse
         Left,       //!< The left mouse button
         Right,      //!< The right mouse button
         Middle,     //!< The middle (wheel) mouse button
-        XButton1,   //!< The first extra mouse button
-        XButton2,   //!< The second extra mouse button
+        XButton1,   //!< The first extra mouse button (not available on Linux)
+        XButton2,   //!< The second extra mouse button (not available on Linux)
```

Comment:
As pointed out by @kimci86, these buttons are available through events, so I guess we might not want to have this here